doc/api/inspector.md

```diff
@@ -0,0 +1,60 @@
+# Inspector
+
+> Stability: 1 - Experimental
+
+The `inspector` module provides an API for interacting with the V8 inspector.
+
+It can be accessed using:
+
+```js
+const inspector = require('inspector');
```

Comment:
If we don't hear back from @AndreasMadsen, I think we should rename the module to avoid conflict with the existing module on npm.